packages/ag-charts-community/src/chart/series/cartesian/barSeriesModule.ts

```diff
@@ -31,6 +31,7 @@ export const BarSeriesModule: SeriesModule<'bar'> = {
             },
         ],
     },
+    tooltipDefaults: { range: 'exact' },
```

Comment:
Do we need a distinct `tooltipDefaults` options, rather than just specifying this in `themeTemplate.series.tooltip`? I think this would also mean we don't need special handling for falling back to defaults?

Reply:
Using `themeTemplate.series.tooltip` would require us to add `series[].tooltip.range` to the options contract and expose this option in the API. This is undesirable; because it would require us to support the API feature indefinitely or removing at some point (breaking change). Using a distinct `tooltipDefaults` option gives us flexibility to remove/change it as we want.

We could perhaps add it into something generalised like `internalThemeTemplate`, but it unclear right now at this whether that's necessary.